docs/reference/avalanchego/API-admin.md

```diff
@@ -1,5 +1,9 @@
 ---
-sidebar_position: 6
+id: admin-api
```

Comment:
A good practice should keep the file name/path consistent so that no id is needed. Introducing id for renaming makes the maintenance not easy.

Reply:
A lot of our files have names that do not coincide with the topic covered, and the links tend to either be inconsistent in format or run-on. 

Using a id in the markdown front matter creates a stable and predictable URL for that specific page, and you ensure that its URL remains consistent even if the page title or file name changes in the future. This also makes it easier to give very descriptive names to the files while keeping links consistent.

Reply:
reconsidering, I've removed id's from the front matter; it is definitely more intuitive for open-source contributors to navigate and raise PRs this way. In the future, I'd like to establish a file naming convention of some sort to enforce a higher standard for our URL/filenaming.